OJP/OJP_Lines.xsd

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://www.vdv.de/ojp" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:siri="http://www.siri.org.uk/siri" xmlns:gml="http://www.opengis.net/gml/3.2" targetNamespace="http://www.vdv.de/ojp" elementFormDefault="qualified" attributeFormDefault="unqualified" id="OJP_siri_service">
+	<xs:annotation>
+		<xs:documentation>OJP/OJP_Lines.xsd - OJP Line information request and response</xs:documentation>
+	</xs:annotation>
+	<xs:import namespace="http://www.siri.org.uk/siri" schemaLocation="../siri/siri_model/siri_all-v2.0.xsd"/>
+	<xs:include schemaLocation="OJP_All.xsd"/>
+	<xs:annotation>
+		<xs:documentation>=============================== Line Information Service ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:annotation>
+		<xs:documentation>=============================== Line Information Response Group ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="LineInformationResponseGroup">
+		<xs:annotation>
+			<xs:documentation>Line Information response group.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="LineResult" type="LineResultStructure" minOccurs="0" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:group>
+	<xs:annotation>
+		<xs:documentation>=============================== LineInformation support functions ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:complexType name="LineResultStructure">
+		<xs:sequence>
+			<xs:element name="LineRef" type="siri:LineRefStructure">
+				<xs:annotation>
+					<xs:documentation>Reference to the LINE.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element ref="siri:PublishedLineName" minOccurs="0"/>
+			<xs:element name="DirectionRef" type="siri:DirectionRefStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>DIRECTION of LINE.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="RouteGeometry" type="LinkProjectionStructure" minOccurs="0">
```

Comment:
Name the type **RouteGeometryStructure** (or **LinearShapeStructure** if you want to be generic), and make it unbounded, and that's Ok for me (with a comment "see NeTEx data for the detailed structure")

Reply:
switched to LinearShapeStructure and unbounded